closed/OpenJ9.gmk

```diff
@@ -86,6 +86,22 @@ endif
 	stage_openj9_build_jdk \
 	#
 
+# jitserver
+
+ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+$(eval $(call SetupCopyFiles,OPENJ9_STAGED_FILES, \
+	DEST := $(JDK_OUTPUTDIR)/bin, \
+	SRC := $(OPENJ9_VM_BUILD_DIR), \
+	FILES := jitserver \
```

Comment:
This should use `$(EXE_SUFFIX)` (here and on line 101).

Reply:
Done